lib/util/memlimit.c

```diff
@@ -63,6 +63,11 @@
 #undef HW_USERMEM
 #endif
 
+/* Is RLIMIT_DATA meaningful on this platform? */
```

Comment:
I would s/meaningful/relevant/.  There may be systems where malloc invokes sbrk and RLIMIT_DATA is *meaningful* but not *relevant* to us because we're using mmap.